CHANGELOG.md

```diff
@@ -7,10 +7,24 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Notes
 
+If you are upgrading from 0.10.5 or earlier you will need to change
+your application (see Notes for 0.11.0 release). For details on API
+changes see [API Changes](https://github.com/oracle/helidon/wiki/API-Changes-in-1.0)
+
+If you are upgrading from 0.11.0 note that media support (for example JsonSupport)
+has moved to a top level `media` component. Details
+in [API Changes](https://github.com/oracle/helidon/wiki/API-Changes-in-1.0). 
+
 ### Improvements
 
+- WebServer: Add JAX-B (Yasson) support [388](https://github.com/oracle/helidon/pull/388)
```

Comment:
Should be `JSONB`.  `JAXB` is the technology for binding XML.

Reply:
Doh!